test/functional/ui/searchhl_spec.lua

```diff
@@ -483,7 +474,15 @@ describe('search highlighting', function()
     -- check hilights work also in folds
     feed("zf4j")
     command("%foldopen")
-    screen:expect_unchanged()
+    screen:expect([[
```

Comment:
This is to avoid the intermediate screen state that's causing the `expect_unchanged()` to fail.